README.md

```diff
@@ -1358,9 +1358,9 @@ Thank you to all our collaborators! 🙏
 
 Our collaborators are members who are contributing to the repository on a regular basis, through suggesting new best practices, triaging issues, reviewing pull requests and more. If you are interested in helping us guide thousands of people to craft better Node.js applications, please read our [contributor guidelines](/.operations/CONTRIBUTING.md) 🎉
 
-| <a href="https://github.com/idori" target="_blank"><img src="assets/images/members/ido.png" width="75" height="75"></a> | <a href="https://github.com/TheHollidayInn" target="_blank"><img src="assets/images/members/keith.png" width="75" height="75"></a> |
-| :---------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------: |
-|                                    [Ido Richter (Founder)](https://github.com/idori)                                    |                                        [Keith Holliday](https://github.com/TheHollidayInn)                                         |
+| <a href="https://github.com/idori" target="_blank"><img src="assets/images/members/ido.png" width="75" height="75"></a> | <a href="https://github.com/TheHollidayInn" target="_blank"><img src="assets/images/members/keith.png" width="75" height="75"></a> | <a href="https://github.com/rluvaton" target="_blank"><img src="assets/images/members/raz-luvaton.png" width="75" height="75"></a> |
+| :---------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------: |
+|                                    [Ido Richter (Founder)](https://github.com/idori)                                    |                                        [Keith Holliday](https://github.com/TheHollidayInn)                                         |                                        [Raz Luvaton (Full Stack Developer)](https://github.com/rluvaton)                                         |
 
 ### Collaborator Emeriti
 
```

Comment:
This comment is about the next file: `assets/images/members/raz-luvaton.png` (cause GitHub doesn't allow commenting on binary files 🤦🏻‍♂️ ).
1. Considering the fact the image is rendered at 75x75, I'd replace that asset with a smaller one.
2. JPG is more suitable for photos and will compress better than PNG.

Reply:
> 1. Considering the fact the image is rendered at 75x75, I'd replace that asset with a smaller one.

I've tried many websites and sizes but the image looked blurry so I kept the image size

> 2. JPG is more suitable for photos and will compress better than PNG.

Done!

Reply:
The 100k JPEG is 7 times smaller than the original PNG.